victron_ble/devices/solar_charger.py

```diff
@@ -0,0 +1,80 @@
+from construct import Int16sl, Int16ul, Struct
+
+from victron_ble.devices.base import Device, DeviceData, OperationMode
+
+
+class SolarChargerData(DeviceData):
+    def get_charge_state(self) -> OperationMode:
```

Comment:
I'm a bit against this, because Victron itself is calling this "State" in their screen...

![IMG_51006A07F7F8-1](https://user-images.githubusercontent.com/8994357/210321717-90bd4d65-599a-4ce4-8509-215452e5b747.jpeg)